app/frontend/src/store/modules/form.js

```diff
@@ -76,11 +109,30 @@ export default {
     updateIpcPlan: (state, obj) => {
       Object.assign(state.ipcPlan, obj);
     },
-    setStep: (state, step) => {
-      window.scrollTo(0, 0);
-      state.step = step;
-    }
   },
   actions: {
+    async submitForm({ commit, state }) {
+      commit('setSubmitting', true);
+      commit('setSubmissionError', '');
+      try {
+        const body = {
+          business: state.business,
+          contacts: state.contacts,
+          ipcPlan: state.ipcPlan
+        };
+        await new Promise(r => setTimeout(r, 1000));
```

Comment:
Not sure what this singleton async delay is for. Probably an artifact of TODO scaffolding and should be removed.

Reply:
yeah exactly